src/meta/base_values.yaml

```
@@ -0,0 +1,514 @@
+minecraft:stripped_oak_log: 20
+minecraft:stripped_spruce_log: 20
+minecraft:stripped_birch_log: 20
+minecraft:stripped_jungle_log: 20
+minecraft:stripped_acacia_log: 20
+minecraft:stripped_dark_oak_log: 20
+minecraft:stripped_crimson_stem: 20
+minecraft:stripped_warped_stem: 20
+minecraft:oak_leaves: 1
+minecraft:spruce_leaves: 1
+minecraft:birch_leaves: 1
+minecraft:jungle_leaves: 1
+minecraft:acacia_leaves: 1
+minecraft:dark_oak_leaves: 1
+minecraft:azalea_leaves: 1
+minecraft:flowering_azalea_leaves: 1
+minecraft:shroomlight: 20
+minecraft:honeycomb: 20
+
+# Plants
+minecraft:glass: 10
+minecraft:cobweb: 10
+minecraft:grass: 1
+minecraft:fern: 1
+minecraft:azalea: 1
+minecraft:flowering_azalea: 1
+minecraft:dead_bush: 1
+minecraft:seagrass: 1
+minecraft:sea_pickle: 1
+minecraft:dandelion: 1
+minecraft:poppy: 1
+minecraft:blue_orchid: 1
+minecraft:allium: 1
+minecraft:azure_bluet: 1
+minecraft:red_tulip: 1
+minecraft:orange_tulip: 1
+minecraft:white_tulip: 1
+minecraft:pink_tulip: 1
+minecraft:oxeye_daisy: 1
+minecraft:cornflower: 1
+minecraft:lily_of_the_valley: 1
+minecraft:wither_rose: 100
+minecraft:spore_blossom: 1
+minecraft:brown_mushroom: 1
+minecraft:red_mushroom: 1
+minecraft:crimson_fungus: 1
+minecraft:warped_fungus: 1
+minecraft:crimson_roots: 10
+minecraft:warped_roots: 10
+minecraft:nether_sprouts: 1
+minecraft:weeping_vines: 1
+minecraft:twisting_vines: 1
+minecraft:sugar_cane: 5
+minecraft:kelp: 5
+minecraft:moss_block: 5
+minecraft:hanging_roots: 5
+minecraft:big_dripleaf: 1
+minecraft:small_dripleaf: 1
+minecraft:bamboo: 5
+minecraft:cactus: 5
+minecraft:pumpkin: 5
+minecraft:carved_pumpkin: 5
+minecraft:red_mushroom_block: 5
+minecraft:mushroom_stem: 5
+minecraft:vine: 5
+minecraft:glow_lichen: 5
+minecraft:mycelium: 1
+minecraft:lily_pad: 10
+minecraft:sunflower: 1
+minecraft:lilac: 1
+minecraft:rose_bush: 1
+minecraft:peony: 1
+minecraft:tall_grass: 1
+minecraft:large_fern: 1
+minecraft:turtle_egg: 50
+minecraft:cocoa_beans: 5
+minecraft:nether_wart: 5
+minecraft:small_amethyst_bud: 5
+minecraft:medium_amethyst_bud: 10
+minecraft:large_amethyst_bud: 20
+minecraft:amethyst_cluster: 100
+minecraft:brown_mushroom_block: 1
+
+# Special building blocks
+minecraft:petrified_oak_slab: -1
+minecraft:smooth_quartz: 40
+minecraft:smooth_red_sandstone: 20
+minecraft:smooth_sandstone: 20
+minecraft:smooth_stone: 4
+minecraft:obsidian: 100
+minecraft:spawner: -1
+minecraft:farmland: 1
+minecraft:ice: 2
+
+# Stone variants
+minecraft:infested_stone: 1
+minecraft:infested_cobblestone: 1
+minecraft:infested_stone_bricks: 1
+minecraft:infested_mossy_stone_bricks: 1
+minecraft:infested_cracked_stone_bricks: 1
+minecraft:infested_chiseled_stone_bricks: 1
+minecraft:infested_deepslate: 1
+minecraft:cracked_stone_bricks: 1
+minecraft:cracked_deepslate_bricks: 1
+minecraft:cracked_deepslate_tiles: 1
+minecraft:cracked_nether_bricks: 1
+
+# End
+minecraft:chorus_plant: 20
+minecraft:chorus_flower: 5
+minecraft:end_stone: 20
```

```suggestion
minecraft:chorus_plant: 10
minecraft:chorus_flower: 1
minecraft:end_stone: 1
```
This seems like too much. If someone has unlocked the end and mines end stone, this will be way overpowered.

Mining end stone with efficiency is *fast*. IIRC chorus fruits and flowers and stuff are really easily obtained as well. They can be slightly higher value since you need to transport the items all the way from an end city, but still not too high.